login.js

```diff
@@ -0,0 +1,75 @@
+const bcrypt = require('bcrypt');
+const { app, databaseConnection, con } = require('./app');
+
+
+const init = async () => {
+
+    try {
+
+        await databaseConnection();
```

Comment:
databaseConnection should be estabilished in app.js itself. not inside login.js as mentioned